extensions/sonarcloud/tasks/buildbreaker/common/checkQualityGateTask.ts

```diff
@@ -51,10 +51,10 @@ export default async function checkQualityGateTask(endpoint: Endpoint) {
 
 async function getTaskReport(): Promise<TaskResult> {
   return new Promise<TaskResult>(resolve => {
-    const taskReportGlob = path.join('**', 'report-task.txt');
+    const taskReportGlob = path.join(tl.getVariable('Build.BuildNumber'), 'report-task.txt');
```

Comment:
I'm not sure that this is a good idea.  The ** is far more flexible.

Reply:
I get that information from the SonarQube plugin code, but '**' is also ok for me. I integrate that in the pull request.